dev/js/Calendar.js

```diff
@@ -60,10 +60,19 @@
 
     this.calendarHTML(this.type);
 
+    $('.daterange').attr('aria-live', 'polite');
+
     $('.dr-presets', this.element).click(function() {
       self.presetToggle();
     });
 
+    $('.dr-presets', this.element).keypress(function(event) {
+      var keycode = event.keyCode || event.which;
```

Comment:
`keyCode` has been deprecated, [see here](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/keyCode). Browser support is still good for it but could be removed at any time.

A good alternative would be to use `event.key`, [see here](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/key).

Reply:
Ahh thanks @Jesswinters !

Reply:
Okay! Based on the MDN page for `keyCode`, it's recommended to use it as a backup to `event.key`. [More info here as well.](https://dev.to/taufik_nurrohman/bringing-keyboardevent-key-and-keyboardevent-keycode-altogether-for-the-best-keyboard-interaction-experience-jlf)